etl/helpers.py

```diff
@@ -3,17 +3,24 @@
 #  etl
 #
 
+import re
 import tempfile
 from contextlib import contextmanager
 from pathlib import Path
-from typing import Any, Iterator, List, cast
+from typing import Any, Dict, Iterator, List, Optional, Union, cast
 
 import requests
 from owid import catalog
+from owid.datautils.common import ExceptionFromDocstring
 from owid.walden import Catalog as WaldenCatalog
 from owid.walden import Dataset as WaldenDataset
 
 from etl import paths
+from etl.snapshot import Snapshot
+from etl.steps import load_dag
+
+# Load ETL dag.
+DAG = load_dag()
```

Comment:
Loading it on module level is ok I think, we're not making dynamic changes to DAG.